bsp/src/mill/bsp/MillBuildServer.scala

```diff
@@ -361,6 +336,50 @@ class MillBuildServer(evaluator: Evaluator, bspVersion: String, serverVersion: S
       new CleanCacheResult(msg, cleaned)
     }
 
+  override def workspaceReload(): CompletableFuture[Object] =
+    handleExceptions {
+      BSP.install(evaluator)
```

Comment:
This seem far from sufficient and would effectively implemented with a redesign of how we run BSP outside of a task command.

Reply:
Just in case it was an oversight, this will never get hit by a client. Without having 
```scala
capabilities.setCanReload(true)
```
in the `buildInitialize` clients probably won't event attempt to trigger this since right now the field is absent, so the assumption is that it's not implemented.

Reply:
Thanks for pointing this out @ckipp01 I was not aware of this.